packages/api/lib/orca.js

```diff
@@ -1,11 +1,42 @@
 'use strict';
 
 const cloneDeep = require('lodash/cloneDeep');
-const { listRequests } = require('@cumulus/api-client/orca');
+const got = require('got');
 
 const Logger = require('@cumulus/logger');
 const log = new Logger({ sender: 'api/lib/orca' });
 
+/**
+ * post request to ORCA
+ *
+ * @param {Object} params
+ * @param {string} params.orcaApiUri - orca endpoint url
+ * @param {string} params.path - path of the request
+ * @param {Object} params.body - body of the request
+ * @returns {Promise<Object>} - resolves to the ORCA return
+ */
+async function postRequestToOrca({ orcaApiUri = process.env.orca_api_uri, path, body }) {
+  if (!orcaApiUri) {
+    const errMsg = 'The orca_api_uri environment variable is not set';
+    log.error(errMsg);
+    throw new Error(errMsg);
+  }
+
+  try {
+    return await got.post(
+      `${orcaApiUri}/${path}`,
+      {
+        json: body || {},
+        responseType: 'json',
+        throwHttpErrors: false,
+      }
+    );
+  } catch (error) {
+    log.error('got encountered error:', error);
```

Comment:
instead of "got encountered error". what about "POST to orca encountered error"?